packages/adapter-vercel/index.js

```diff
@@ -446,7 +446,7 @@ function static_vercel_config(builder, config, dir) {
 
 	for (const [src, redirect] of builder.prerendered.redirects) {
 		prerendered_redirects.push({
-			src,
+			src: src.replace(/\/?$/, '/?'),
 			headers: {
 				Location: redirect.location
```

Comment:
Is there any chance that the redirect location is the same except the trailing slash, in which case this would create an infinite loop? Maybe we should double-check that before applying the `/?` replacement

Reply:
Super edge case, since you can't create a `/foo` route alongside a `/foo/` route (best you could do is `/foo` alongside `/[blah]/`, but one would match over the other). But I guess you _could_ do something wacky with a `handle` hook or a custom server, and it does indeed create an infinite loop. Pushed some code to handle that case